vsm

```diff
@@ -118,7 +147,7 @@ class State(object):
 
 
 def emit(signal, value):
-    print('{} = {}'.format(signal, value))
+    Log.i('{} = {}'.format(signal, value))
```

Comment:
It currently doesn't break since timestamps are currently off. But we'll eventually need them for the log diff tool which is supposed to factor them in. We could enable/disable them with a command-line argument to `vsm`.

What's your preference?